r6sapi/definitions/converters.py

```diff
@@ -0,0 +1,221 @@
+"""
```

Comment:
The name of this file seems a bit misleading to me - reading the code it seems more like a collection of data class stores rather than converters? Maybe `stores.py` would better describe its function?

Reply:
True! I'll switch it over.

Reply:
Actually I plan to split out the converting functionality to a new file; and have the new `stores.py` simply store `Info` objects.